packages/inventory/src/EntityTableToolbar.js

```diff
@@ -1,61 +1,108 @@
-import React from 'react';
+/* eslint-disable no-unused-vars */
+import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
-import { Filter } from './Filter';
-import { TableToolbar, Skeleton, SkeletonSize } from '@redhat-cloud-services/frontend-components';
-import { Split, SplitItem } from '@patternfly/react-core';
-
-const EntityTableToolbar = ({
-    total,
-    page,
-    onRefresh,
-    perPage,
-    filters,
-    hasItems,
-    pathPrefix,
-    apiBase,
-    children,
-    pagination,
-    loaded,
-    ...props
-}) => (
-    <TableToolbar className="ins-c-inventory__table--toolbar">
-        <Split>
-            <SplitItem>
-                <Filter { ...props }
-                    hasItems={ hasItems }
-                    filters={ filters }
-                    pathPrefix={ pathPrefix }
-                    apiBase={ apiBase }
-                    totalItems={ total }
-                />
-            </SplitItem>
-            <SplitItem isFilled>
-                { children }
-            </SplitItem>
-            <SplitItem>
-                {
-                    loaded ?
-                        pagination :
-                        <Skeleton size={ SkeletonSize.lg } />
-                }
-            </SplitItem>
-        </Split>
-    </TableToolbar>
-);
+import { Skeleton, SkeletonSize, PrimaryToolbar } from '@redhat-cloud-services/frontend-components';
+import debounce from 'lodash/debounce';
+
+const TEXT_FILTER = 'hostname_or_id';
+
+class EntityTableToolbar extends Component {
+    state = {
+        textFilter: ''
+    }
+
+    debouncedRefresh = debounce((config) => {
+        this.props.onRefresh && this.props.onRefresh(config);
+    }, 800);
```

Comment:
is that 800ms pause between typing? is that typical for a typeahead / autocomplete solution (I have no idea)?

https://ux.stackexchange.com/questions/36231/when-should-you-start-autocompleting-a-search-in-autocomplete-search-field?rq=1

Reply:
i realize that we can always tweak this later...

Reply:
Yes, the 800 is in milliseconds between requests to server. If user does not type anything in this window request is sent. On the other hand if user is still typing no new requests are sent to server. In the past we chose 800 as the base for this, however we should perhaps add this number to utils constant in order to make this persistent accross entire platform - https://github.com/RedHatInsights/frontend-components/issues/197